htdocs/core/js/lib_notification.js.php

```diff
@@ -84,10 +84,14 @@ function check_events() {
                             var url="notdefined";
                             var title="Not defined";
                             var body = value['tipo'] + ': ' + value['titulo'];
-                            if (value['type'] == 'agenda' && value['location'] != null && value['location'] != '') {
+                            if (value['type'] == 'agenda' && (value['location'] != null || value['location'] != '')) {
```

Comment:
The test 
(value['location'] != null || value['location'] != '')
seems always true
&& seems the good choice